images/bridge2/assistant/assistant.go

```diff
@@ -109,24 +110,22 @@ Overall {} is a powerful system that can help humans with a wide range of tasks
 `, "{}", name)
 }
 
-func (a *OpenAIClient) Call(speaker, prompt string) (string, error) {
+func (a *OpenAIClient) Call(speaker, input string) (string, error) {
 	maxTokens := 4096
 	systemMessage := a.SystemMessage
-	req := ChatCompletionRequest{
+	prompt, err := prompts.Render("complete", map[string]any{
+		"SystemMessage": systemMessage,
+		"UserInput":     input,
+	})
+	if err != nil {
+		return "", err
+	}
+	req := CompletionRequest{
+		// Model:
```

Comment:
Is there a value we want here for the model?

Reply:
no, nothing specific atm